scripts/appleseedMaya/aetemplate/NEasLuminanceTemplate.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<templates>
+	<using package='maya'/>
```

Comment:
@dictoon it's time to add XML autodetect and configuration to vim, by default it seems it sets tabstop to 8. Fixed.